resources/views/layout.blade.php

```diff
@@ -55,42 +55,53 @@
                                 </ul>
                             </li>
-                            <li class="col-sm-6">
+                        </ul>
+                    </li>
+                </ul>
+
+                <ul class="nav navbar-nav navcustom">
+                    <li class="dropdown dropdown-large">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community <b class="caret"></b></a>
+                        <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
+                            <li class="col-sm-12">
                                 <ul>
+                                    <li class="dropdown-header">Membership</li>
+                                    <li>{!! link_to_route("visiting.landing", "Visit the UK") !!}</li>
+                                    <li>{!! link_to_route("visiting.landing", "Transfer to the UK") !!}</li>
+                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
+                                    <li class="divider"></li>
                                     <li class="dropdown-header">Third-Party Services</li>
                                     <li>{!! link_to_route("teamspeak.new", "TS Registration") !!}</li>
                                     <li>{!! link_to_route("slack.new", "Slack Registration") !!}</li>
-                                    <li class="divider"></li>
-                                    <li class="dropdown-header">Community Features</li>
-                                    <li>{!! link_to_route("mship.notification.list", "View Notifications") !!}</li>
-                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
-                                    <li>{!! link_to_route("mship.feedback.new", "Controller/Pilot Feedback") !!}</li>
                                 </ul>
                             </li>
                         </ul>
-
                     </li>
                 </ul>
 
-                @if(Auth::check())
-                    <ul class="nav navbar-nav navcustom">
+                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                    {!! Form::open(['route' => 'logout', 'id' => 'logout-form']) !!}
+                    <ul class="nav navbar-nav navcustom navbar-right">
                         <li class="dropdown dropdown-large">
                             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Account <b class="caret"></b></a>
-                            <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
-                                <li class="col-sm-6">
+                            <ul class="dropdown-menu dropdown-menu-logout dropdown-menu-large row mainmenu_dropdown">
+                                <li class="col-sm-12">
                                     <ul>
```

Comment:
My bad, completely forgot to consider longer names. I'll fix that when I'm back after the weekend. I could fit the admin panel in somewhere, I was thinking the resources bar as I'm not sure it would fit anywhere else, unless it should have a separate button. Would that work?